packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop                      | Type                                                       | Default     | Description                                                                    | Required |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children                  | `number`                                                   | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent                    | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                       |          |
+| fontWeight                | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                        |          |
+| suffix                    | `true`, `false`                                            | `false`     | Indicates whether a suffix should be used                                      |          |
```

Comment:
What about the `showCurrencySuffix` from earlier? Just `suffix` sounds vague.

Reply:
changed it to `hasSuffix`.

Reply:
Can we use `showSuffix`? Generally we're following `show*` terminology in other components.

Reply:
cool, done.